model/task/task.go

```diff
@@ -103,6 +103,9 @@ type Task struct {
 	// tasks
 	HostId string `bson:"host_id" json:"host_id"`
 
+	// The version of the agent this task was run on.
+	AgentVersion string `bson:"agent_version" json:"agent_version"`
```

Comment:
I think this should be omitempty. There are lots of tasks, many of which won't run. This prevents task documents from getting unnecessarily huge.